lib/retriable.rb

```diff
@@ -59,13 +62,23 @@ def retriable(opts = {})
       begin
         return Timeout.timeout(timeout) { return yield(try) } if timeout
         return yield(try)
-      rescue *[*exception_list] => exception
+      rescue *[*rescue_list] => exception
```

Comment:
If we put `on` and `ignore` in an array and flatten it, do we still need this array splat expansion? This is here in case `on` is not an array, but you're with your changes on line 41-42, it's impossible now for this to not be an array.

Reply:
Yes, my mistake. Updating.